internal/controllers/utils/constants.go

```diff
@@ -68,13 +68,19 @@ const (
 	defaultServiceCAFile    = "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt" // nolint: gosec // hardcoded path only
 )
 
+// Required template schema parameters
+const (
+	TemplateParamNodeClusterName = "nodeClusterName"
+	TemplateParamOCloudSiteId    = "oCloudSiteId"
+	TemplateParamClusterInstance = "clusterInstanceParameters"
+	TemplateParamPolicyTemplate  = "policyTemplateParameters"
```

Comment:
Maybe rename `TemplateParamPolicyTemplate` to `TemplateParamPolicyConfig`? I find it a bit weird to have `template` twice in the variable name.